web/src/components/data-fields/non-essential-questions/NonEssentialQuestion.tsx

```diff
@@ -28,7 +29,9 @@ export const NonEssentialQuestion = (props: Props): ReactElement => {
       {nonEssentialQuestionText && (
         <>
           <div className={"margin-top-2"}>
-            <span className={"text-bold"}>{nonEssentialQuestionText}</span>
+            <span className={"text-bold"}>
+              <Content>{nonEssentialQuestionText}</Content>
+            </span>
```

Comment:
[sand] I know it's mostly meaningless these days, but since `<Content>` can include block-level elements, it should be wrapped in a block-level element rather than a `<span>`.